src/man/sssd-ad.5.xml

```diff
@@ -109,6 +109,21 @@ ldap_id_mapping = False
             case-insensitive in the AD provider for compatibility with Active
             Directory's LDAP implementation.
         </para>
+        <para>
+            SSSD only resolves Active Directory Security Groups. For more
+            information about AD group types see:
+            <ulink
+            url="https://docs.microsoft.com/en-us/windows-server/identity/ad-ds/manage/understand-security-groups">
+            Active Directory security groups</ulink>
+        </para>
+        <para>
+            SSSD filters out Domain Local groups from remote domains in the AD
+            forest. By default they are filtered out e.g. when following a
+            nested group hierarchy in remote domains because they are not valid
+            in the local domain. This behavior is dependent on the value for
+            the <quote>ad_allow_remote_domain_local_groups</quote> option
```

Comment:
Hi,

thanks for the update but please remove the last sentence. This option should not get any additional attention. And I would like to ask you to add a sentence like "This is done to be in agreement with Active Directory's group-membership assignment which can e.g. be seen in the PAC of the Kerberos ticket of a user issued by Active Directory."

bye,
Sumit

Reply:
Done, thanks.